packages/ui-color-picker/src/ColorPreset/__tests__/ColorPreset.test.tsx

```diff
@@ -359,14 +353,5 @@ describe('<ColorPreset />', () => {
     })
   })
 
-  // TODO: this block throws a ton of errors, we should investigate
-  // describe('should be accessible', () => {
-  //   generateA11yTests(ColorPreset, ColorPresetExamples)
-  //   it('a11y', async () => {
-  //     await mount(<ColorPreset {...testValue} />)
-  //     const subject = await ColorPresetLocator.find()
-  //
-  //     expect(await subject.accessible()).to.be.true()
-  //   })
-  // })
+  // The accessibility tests are ignored because the tooltips of the ColorIndicator, which are defined in the "aria-labelledby" attribute, are located out of the scope of the ColorPreset.
```

Comment:
I checked this component with the a11y checker tool, and it indeed throws tons of errors... Well, the good news is that all these errors are coming from the fact that we are nesting the drilldown and tooltip components, the same element is the trigger for both, so we should look for a workaround here...

@HerrTopi what did the a11y audit say? I'm pretty sure they found these errors too, but were they willing to ignore them?

<img width="1043" alt="image" src="https://user-images.githubusercontent.com/11446033/198621025-b9b94cb1-2220-4bfd-a624-e098913330f5.png">

Reply:
I honestly don't remember, but I think we should ask for a new audit anyways